app/upgrades/v28/upgrades.go

```diff
@@ -56,3 +78,29 @@ func UpdateRedemptionRateBounds(ctx sdk.Context, k stakeibckeeper.Keeper) {
 		k.SetHostZone(ctx, hostZone)
 	}
 }
+
+func DeliverLockedTokens(ctx sdk.Context, ak authkeeper.AccountKeeper, bk bankkeeper.Keeper) error {
+	// Get account
+	from := sdk.MustAccAddressFromBech32(FromAccount)
+	to := sdk.MustAccAddressFromBech32(DeliveryAccount)
+	amt := sdk.NewCoins(sdk.NewInt64Coin("ustrd", LockedTokenAmount))
+
+	// Destination must exist and be a plain BaseAccount.
+	base, isBaseAccount := ak.GetAccount(ctx, to).(*authtypes.BaseAccount)
+	if !isBaseAccount {
+		// Maybe return an error here?
```

Comment:
I think the tradeoff is that in the chance the upgrade fails:
- Return error = chain's halted until validators can apply a patch
- Return nil = no downtime but need to circulate a second upgrade
I think I slightly lean towards returning no error, but could go either way